app/assets/stylesheets/custom.css.scss

```diff
@@ -135,6 +130,18 @@ h1 {
   background-color: darkgrey;
 }
 
+#flash_twitter {
```

Comment:
Avoid using id selectors<br>Selector `flash_twitter` should be written in lowercase with hyphens